pygsti/models/memberdict.py

```diff
@@ -45,18 +45,6 @@ def __setitem__(self, key, val):
                            "beginning with the prefix '%s'" % self._prefix)
         super(_PrefixOrderedDict, self).__setitem__(key, val)
 
-    #Handled by derived classes
-    #def __reduce__(self):
-    #    items = [(k,v) for k,v in self.iteritems()]
-    #    return (_PrefixOrderedDict, (self._prefix, items), None)
-
-    """
```

Comment:
Remove reduce, but leave docstring

Reply:
@sserita are you sure? The docstring doesn't seem to apply to this class. It applies somewhat to the class below, but that class has more detailed docs.